.github/workflows/nightly_benchmarks.yml

```
@@ -0,0 +1,83 @@
+name: Nightly Benchmarks
+
+on:
+  schedule:
+    # Run every day at midnight
+    - cron: "0 0 * * *"
+
```

```suggestion
  workflow_dispatch

```
Let's also add `workflow_dispatch` then we can push a button in the GitHub interface to run the action.